src/include/storage/table/rel_table.h

```diff
@@ -157,6 +157,9 @@ class KUZU_API RelTable final : public Table {
     void update(transaction::Transaction* transaction, TableUpdateState& updateState) override;
     bool delete_(transaction::Transaction* transaction, TableDeleteState& deleteState) override;
 
+    // Deletes all edges attached to the node(s) specified in the deleteState
+    // Currently only supports deleting from a single src node
```

Comment:
I don't think we should say "currently only". the meaning of this interface should be to delete outgoing rels of given nodes in each storage direction.